scripts/setup_web1.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Let's create this as a gist and send to @abezella . Then maybe add a link to that gist on https://github.com/internetarchive/openlibrary/wiki/Production-Service-Architecture

Reply:
i don't think i need to be in the workflow for this(?)  the deployment of the services should be in the hands of the openlibrary team...

Reply:
Should this be in Ansible somewhere??

Reply:
it can go into ansible or whatever you choose to manage deployments.  i'd encourage ansible as it is what the rest of the archive seems to be settling on

Reply:
Last time I spoke w/ @bfalling we had a pretty hard requirement on doing things the same way as IA core team and that meant Ansible.

Not sure where everything will live eventually, but for now the process according to Andy is:

> you can provision from any host that has ansible installed. so (to use that playbook) you would checkout the olsystem repo, then run something like:

```
  ansible-playbook --become --inventory="playbooks/inventory" --limit="ol-web1" "playbooks/site.yml"
```

```
--become tells it to sudo after ssh'ing in
--limit can be a single or a range of hosts defined in the inventory file (ol-web[134]) or a group defined in the inventory (memcache) or a list separated by : (=or) or :& (=and)
```